test/server/test_image_validation.py

```diff
@@ -77,7 +77,61 @@ def test_invalid_data_url():
         assert resp.status_code == 500
 
 
+def test_invalid_data_jpeg_image():
+    image = image_setup()
```

Comment:
`image_setup` is returning a data url for a png, which won't work for JPEG tests.